src/components/report-problem/report-problem.component.js

```diff
@@ -38,7 +38,7 @@ export default class ReportProblem extends Component {
         }
       );
 
-      this.props.saveAnnotationActionCreator(geoHash, {'lat': this.props.userLocation.lat, 'lng': this.props.userLocation.lng});
+      this.props.saveAnnotationActionCreator(geoHash, {'lat': this.props.userLocation.lat, 'lng': this.props.userLocation.lng, 'geoHash': geoHash});
```

Comment:
you should be able to abbreviate this a little to just:

```
this.props.saveAnnotationActionCreator(geoHash, {lat: this.props.userLocation.lat, lng: this.props.userLocation.lng, geoHash});
```

Drop the single quotes and make the last value just `geoHash`